### PR TITLE
[12.0][IMP] fieldservice

### PR DESCRIPTION
**Pull request by murtuzasaleh**
Added fields in order equipment view.

#285

### CODE REVIEW

**Comment by murtuzasaleh** [comment]
@max3903 Can you please review this PR.

File: fieldservice/models/res_config_settings.py [hunk 1]
```diff
@@ -63,6 +66,15 @@ class ResConfigSettings(models.TransientModel):
         related='company_id.auto_populate_persons_on_location',
         readonly=False)
 
+    @api.onchange('group_fsm_equipment')
+    def _onchange_group_fsm_equipment(self):
+        self.auto_populate_the_equipments = self.group_fsm_equipment
```
Comment (max3903) [inline fieldservice/models/res_config_settings.py]: @murtuzasaleh 

If `group_fsm_equipment` is true, `auto_populate_the_equipments` can still be true or false.

**Comment by max3903** [comment]
@murtuzasaleh We need the multi-company rule on `fsm.stage`. The same as in:

https://github.com/OCA/field-service/pull/300/files#diff-0adf37f84aa81782f2c3da512410f7ca

File: fieldservice/models/res_config_settings.py (hunk 1 above)
Comment (murtuzasaleh) [reply fieldservice/models/res_config_settings.py]: @max3903 Improved code, please check it again.

**Comment by murtuzasaleh** [comment]
> @murtuzasaleh We need the multi-company rule on `fsm.stage`. The same as in:
> 
> https://github.com/OCA/field-service/pull/300/files#diff-0adf37f84aa81782f2c3da512410f7ca

Done

**Comment by max3903** [comment]
@murtuzasaleh We are missing the multi-company rule (`ir.rule`).

**Comment by murtuzasaleh** [comment]
> @murtuzasaleh We are missing the multi-company rule (`ir.rule`).

I added it in [fsm.stage multi company rule](https://github.com/OCA/field-service/blob/1f705097865dcc61bbb66b589e9c4db3152509a8/fieldservice/security/res_groups.xml#L24) here.
I will create a new file <'security/ir_rule.xml'> and move it here.

**Review by max3903 — APPROVED** [review]

**Review by osi-scampbell — APPROVED** [review]

File: fieldservice/models/res_config_settings.py [hunk 2]
```diff
@@ -63,6 +65,24 @@ class ResConfigSettings(models.TransientModel):
         related='company_id.auto_populate_persons_on_location',
         readonly=False)
 
+    def get_values(self):
+        res = super(ResConfigSettings, self).get_values()
+        res.update(auto_populate_the_equipments=self.env['ir.config_parameter'].sudo().
```
Comment (osi-scampbell) [inline fieldservice/models/res_config_settings.py]: @murtuzasaleh LGTM can we just get line 70 shortened? Travis build still has red light

**Pull request merged**